store/copr/coprocessor.go

```diff
@@ -977,6 +977,10 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *tikv.R
 			zap.Uint64("regionID", task.region.GetID()),
 			zap.String("storeAddr", task.storeAddr),
 			zap.Error(err))
+		if strings.Contains(err.Error(), "write conflict") {
```

Comment:
Do we need add a limitation that is a "Rc Check Ts read" ? I mean whether some other scenes make a "write conflict" here.